tests/integration/test_sqs.py

```diff
@@ -690,6 +690,57 @@ def test_lambda_invoked_by_sqs_message_with_delay_seconds_dotnetcore31(self):
             zip_file, handler, LAMBDA_RUNTIME_DOTNETCORE31
         )
 
+    def _run_test_fifo_queue_send_multiple_messages(self):
+        fifo_queue = 'my-queue.fifo'
+
+        message_group = 'group-%s' % short_uid()
+        message_dedup = 'dedup-%s' % short_uid()
+        results = []
+        number_of_messages = 5
+
+        queue_url = self.client.create_queue(QueueName=fifo_queue, Attributes={'FifoQueue': 'true'})['QueueUrl']
+
+        # it should preserve .fifo in the queue name
+        self.assertIn(fifo_queue, queue_url)
+
+        # try sending multiple message with message group ID and deduplication ID
+        for i in range(number_of_messages):
+            rs = self.client.send_message(QueueUrl=queue_url, MessageBody='message-{}'.format(i),
+                                          MessageDeduplicationId=message_dedup, MessageGroupId=message_group)
```

Comment:
One thing I noticed in issue #2710 is that a different deduplication ID is used for each message - would that make a difference in this test, potentially?